dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -38,7 +38,7 @@ chill_2.13/0.10.0//chill_2.13-0.10.0.jar
 commons-cli/1.6.0//commons-cli-1.6.0.jar
 commons-codec/1.16.1//commons-codec-1.16.1.jar
 commons-collections/3.2.2//commons-collections-3.2.2.jar
-commons-collections4/4.4//commons-collections4-4.4.jar
+commons-collections4/4.5.0-M1//commons-collections4-4.5.0-M1.jar
```

Comment:
Is `M1` stable?

Reply:
Okay, let's wait for the `official` version to be released. 😄

Reply:
Thank you!